autosubmit/platforms/psplatform.py

```diff
@@ -57,16 +42,23 @@ def __init__(self, expid, name, config):
         self.job_status['FAILED'] = []
         self.update_cmds()
 
+    def get_checkAlljobs_cmd(self, jobs_id):
+        pass
+
+    def parse_Alljobs_output(self, output, job_id):
+        pass
+
+    def parse_queue_reason(self, output, job_id):
+        pass
+
```

Comment:
Moved functions to after the constructor -- can cover it with some tests, but looks silly (and we might not need these functions in the future depending on how we re-design it -- with abc/use parent function/etc.